webapp/src/components/backstage/playbook_list.tsx

```diff
@@ -208,7 +213,7 @@ const PlaybookList = (props: { firstTimeUserExperience?: boolean }) => {
                 onEdit={() => edit(p.id)}
                 onRestore={() => openConfirmRestoreModal({id: p.id, title: p.title})}
                 onArchive={() => openConfirmArchiveModal(p)}
-                onDuplicate={() => duplicatePlaybook(p.id)}
+                onDuplicate={() => handleDuplicatePlaybook(p)}
```

Comment:
Async-await is perfect for this, no need to change. I just couldn't leave this syntactic/literary alternative unmentioned 😜+🤓  ("duplicate playbook then refresh LHS")

```suggestion
                onDuplicate={() => duplicatePlaybook(p.id).then(refreshLHS)}
```